Sent upstream.
fdlibm should provide definition for u_int32_t and u_int64_t
https://bugzilla.mozilla.org/show_bug.cgi?id=1350355

--- firefox-51.0b14/modules/fdlibm/src/math_private.h.orig	2017-01-24 08:42:08.947821459 +0000
+++ firefox-51.0b14/modules/fdlibm/src/math_private.h	2017-01-24 08:50:42.558858228 +0000
@@ -38,7 +38,7 @@
  * endianness at run time.
  */
 
-#ifdef WIN32
+#if 1
 #define u_int32_t uint32_t
 #define u_int64_t uint64_t
 #endif
